spec/features/course/course_user_profile_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Courses: CourseUser Profile' do
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let(:course) { create(:course) }
+    let(:course_student) { create(:course_student, :approved, course: course) }
+    let!(:achievement) { create(:course_user_achievement, course_user: course_student).achievement }
+    let(:course_teaching_assistant) do
+      create(:course_teaching_assistant, :approved, course: course)
+    end
+
+    before { login_as(user, scope: :user) }
+
+    context 'As a Course Teaching Assistant' do
+      let(:user) { course_teaching_assistant.user }
+
+      context "when viewing a student's profile" do
+        let(:course_user) { course_student }
+
+        scenario 'I can see his experience points and achievements' do
+          visit course_course_user_path(course, course_user)
+
+          expect(page).to have_text(course_user.name)
+          expect(page).to have_text(I18n.t('course.course_users.show.achievement_count'))
+          expect(page).to have_link(nil, href: course_achievement_path(course, achievement))
+          expect(page).
+            to have_text(I18n.t('course.course_users.show.experience_points_earned_html'))
+        end
+      end
+    end
+
+    context 'As a Course Student' do
+      let(:student_user) { create(:course_student, :approved, course: course).user }
+
+      context "when viewing a staff's profile" do
+        let(:user) { student_user }
```

Comment:
actually this is a feature spec,. there should not be contexts. it should all be in the scenario as a user story.